Doc/includes/shoddy.c

```diff
@@ -17,7 +17,7 @@ Shoddy_increment(Shoddy *self, PyObject *unused)
 static PyMethodDef Shoddy_methods[] = {
     {"increment", (PyCFunction)Shoddy_increment, METH_NOARGS,
      PyDoc_STR("increment state counter")},
-    {NULL,	NULL},
+    {NULL,      NULL},
```

Comment:
This is part of the documentation. Maybe use a single space? The tab doesn’t seem to align with anything.

Or even drop the second NULL. Other PyMethodDef arrays on the page have a single NULL. The other three fields get zeroed implicitly, and it looks like the first field (_ml_name_) is the one that is checked for NULL.

Reply:
I prefer to drop the second NULL.